test/watcher_test.js

```diff
@@ -199,7 +202,7 @@ describe('Watcher', function() {
 
     it('does nothing if already quitting', function() {
       const adapterQuit = sinon.spy(adapter, 'quit');
-      const watcher = new Watcher(builder, { watcherAdapter: adapter });
+      const watcher = new Watcher(builder, [watchedNodeBasic], { watcherAdapter: adapter });
       watcher._quittingPromise = true;
```

Comment:
using private API in tests is an anti pattern, let's try and use public API's

Reply:
Sure thing